src/cpp/transport/TCPTransportInterface.cpp

```diff
@@ -767,7 +643,8 @@ bool TCPTransportInterface::OpenInputChannel(
     return success;
 }
 
-void TCPTransportInterface::perform_rtcp_management_thread(TCPChannelResource *p_channel_resource)
+void TCPTransportInterface::perform_rtcp_management_thread(
+        std::shared_ptr<TCPChannelResource> channel)
```

Comment:
According to the style guide, one-parameter methods should be declared in one line if doesn't surpass the limit of 120 characters.